From 44bbdbe8702a9d15d0421058dd346c93265cce96 Mon Sep 17 00:00:00 2001
From: Amelie Delaunay <amelie.delaunay@foss.st.com>
Date: Mon, 4 Jan 2021 15:20:45 +0100
Subject: [PATCH 07/88] dmaengine: stm32-mdma: fix
 STM32_MDMA_VERY_HIGH_PRIORITY value

STM32_MDMA_VERY_HIGH_PRIORITY is b11 not 0x11, so fix it with 0x3.

Upstream-Status: Backport [e1263f9277bad198c2acc8092a41aea1edbea0e4]
Signed-off-by: Amelie Delaunay <amelie.delaunay@foss.st.com>
Link: https://lore.kernel.org/r/20210104142045.25583-1-amelie.delaunay@foss.st.com
Signed-off-by: Vinod Koul <vkoul@kernel.org>
---
 drivers/dma/stm32-mdma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/stm32-mdma.c b/drivers/dma/stm32-mdma.c
index f916ba1aa3671..3808db6d347a3 100644
--- a/drivers/dma/stm32-mdma.c
+++ b/drivers/dma/stm32-mdma.c
@@ -198,7 +198,7 @@
 #define STM32_MDMA_MAX_CHANNELS		32
 #define STM32_MDMA_MAX_REQUESTS		256
 #define STM32_MDMA_MAX_BURST		128
-#define STM32_MDMA_VERY_HIGH_PRIORITY	0x11
+#define STM32_MDMA_VERY_HIGH_PRIORITY	0x3
 
 #define STM32_DMA_SRAM_GRANULARITY	PAGE_SIZE
 
-- 
2.42.0

